Kconfig.sof

```diff
@@ -207,6 +207,13 @@ config DSP_RESIDENCY_COUNTERS
 	  R0, R1, R2 are abstract states which can be used differently
 	  based on platform implementation.
 
+config MODULES
```

Comment:
We really need to namespace these as '''MODULES''' is quite generic and may collide with Zephyr.